workflow/kf-rnaseqv4-wf.cwl

```diff
@@ -0,0 +1,878 @@
+cwlVersion: v1.2
+class: Workflow
+id: kfdrc-rnaseq-workflow-v4
+label: Kids First DRC RNAseq Workflow Version 4
+doc: |-
+  # Kids First RNA-Seq Workflow V4
```

Comment:
I suppose this is where I protest the use of version in file names/ids. Maybe we should do Star Trek names:
- The Original Pipeline
- The Next Pipeline
- Deep Space PipeNine

Reply:
Kids First DRC RNAseq TNG Workflow?
Kids First DRC RNAseq Phantom Menace - to follow the STAR WARS release order and boggle the mind